src/annotator/test/frame-observer-test.js

```diff
@@ -0,0 +1,42 @@
+import { findFrames } from '../frame-observer';
```

Comment:
Hmm, we're missing unit tests for `FrameObserver`. Is that currently covered only by an integration test?

Reply:
Yes, only integration test.

Reply:
I added unit tests for `FrameObserver`.